schema/index.go

```diff
@@ -27,12 +27,17 @@ type IndexOption struct {
 
 // ParseIndexes parse schema indexes
 func (schema *Schema) ParseIndexes() map[string]Index {
-	var indexes = map[string]Index{}
-
+	indexes := make(map[string]Index, len(schema.Fields))
 	for _, field := range schema.Fields {
 		if field.TagSettings["INDEX"] != "" || field.TagSettings["UNIQUEINDEX"] != "" {
 			for _, index := range parseFieldIndexes(field) {
-				idx := indexes[index.Name]
+				idx, ok := indexes[index.Name]
+				if !ok {
+					indexes[index.Name] = Index{
+						Fields: make([]IndexOption, 0, 10), // Fields cap default is 10
+					}
+				}
```

Comment:
没懂这个函数修改的目的

Reply:
If map exists, if you don't judge it here, if it doesn't exist, IDX uses zero value, so I add judgment logic. In addition, I have pre allocated field field to avoid expanding slice type in the future. The bottom sort I also made a judgment, when the length is greater than or equal to 2, I do the sorting operation.